package.json

```diff
@@ -60,14 +59,15 @@
     "react-dropzone": "^14.2.2",
     "react-error-boundary": "^3.1.4",
     "react-shadow": "^19.0.3",
-    "spc-parser": "^0.5.2",
+    "spc-parser": "^0.6.0",
```

Comment:
Please keep version 0.5.2. The new version has breaking changes.

Reply:
okey